README.md

```diff
@@ -13,6 +13,7 @@ This module is a general base layer module for setting up a newly provisioned ac
 - IAM Account Settings
 - Activity Tracker routing + COS instance and bucket
 - Trusted Profile + Access Group for Projects
+- CBR Rules + Zones
```

Comment:
Need to add more details here as to let user know what is being denied by default by the CBR rules, reference can be taken from [Pre-wired CBR configuration for FS Cloud](https://github.com/terraform-ibm-modules/terraform-ibm-cbr/tree/main/modules/fscloud#pre-wired-cbr-configuration-for-fs-cloud) doc.